Dom

harshvachhani1 opened the pull request.

Commit:
build basic button file for use js

Commit:
explain todos remove using event

Commit:
write basic notes

Commit:
explain todos data print

Commit:
build notes for searching

Commit:
explain how to rendering dataand find

Commit:
basic code use for search

Commit:
explain for filter in array

Commit:
write code for checkbox

Commit:
explain checkbox click data show and hide

Commit:
some mistake change

Commit:
change link path

Commit:
insert button not remove other

Commit:
change target a tag

Commit:
build form for use search text

Commit:
explain insert data and search text in the array

Commit:
explain checkbox and new add todo

Commit:
explain data saving in local storage

vivek-kachhadiya13 commented on DOM/todo-insert-data/todo-data.js:
```diff
@@ -0,0 +1,64 @@
+
+const renderTodos = (todos, filters) => {
+  const filteredTodos = todos.filter((todo) => {
+    return todo.text.toLowerCase().includes(filters.searchText.toLowerCase());
```
Didn't get this condition. here `todo.text.toLowerCase()` is array? Please verify once i think you will not get array here.

vivek-kachhadiya13 commented on DOM/rendering-data/index.html:
```diff
@@ -0,0 +1,15 @@
+    <h2>Take notes and never forget</h2>
+    <input type="search-text" id="search-text" />
+    <div id="notes"></div>
+    <script src="renderind-data.js"></script>
```
Script should be added in head.

harshvachhani1 replied:
@v

harshvachhani1 replied:
@vi

harshvachhani1 replied:
@vivek-kachhadiya13 bhai noted.

Commit:
apply uuid and build checkbox and textbox

Commit:
explain data search, add new todo and hide used with checkbox

Commit:
explain how to work add, hide checkbox true or false,filter and save localstorage

Commit:
build form used with text,dropdown and create note

Commit:
connect with index file add new note and modify

Commit:
explain how to create note,remove notes,savenotes and applyed filter

Commit:
explain make a array and new value push and create note and filter in notes

Commit:
explain in this file time and when notes update show and depend on edit file

Commit:
build text,checkbox and new add for todos

Commit:
explain to searchtext,new add todo and hide and show function build

Commit:
explain how to work function
make a function to use store localstorage,save todos,remove todos,hide & show and remove,check true or false etc

Commit:
change line no 70 e to declare with name

vivek-kachhadiya13 commented on DOM/edit-note-page/notes-functions.js:
```diff
@@ -0,0 +1,103 @@
+    });
+  } else if (sortBy === "alphabetical") {
+    return notes.sort((fistValue, secondValue) => {
+      if (fistValue.title.toLowerCase() < secondValue.title.toLowerCase()) {
```
Can be written as  ```return fistValue.title.toLowerCase() < secondValue.title.toLowerCase() ? -1 : 0```;
